File: BackEnd/Business Model/ShowModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TVMazeSearch
{
    public class ShowModel
    {
        public int id { get; set; }

        public string url { get; set; }

        public RatingModel rating { get; set; }

        public ImageModel image { get; set; }

        public string name { get; set; }

        public string officialSite { get; set; }
    }
}

File: FrontEnd/src/app/models/show.model.ts
import { ImageModel } from './image.model';
import { RatingModel } from './rating.model';
export class ShowModel {
    public constructor(
        public id:number,
        public url:string,
        public rating:RatingModel,
        public image:ImageModel,
        public name:string,
        public officialSite:string,
        ){}
    
}